gui/src/2D/controls/inputTextArea.ts

```diff
@@ -0,0 +1,1282 @@
+import { Nullable } from "babylonjs/types";
+import { Observable } from "babylonjs/Misc/observable";
+import { Vector2 } from "babylonjs/Maths/math.vector";
+
+import { Control } from "./control";
+import { ValueAndUnit } from "../valueAndUnit";
+import { VirtualKeyboard } from "./virtualKeyboard";
+import { RegisterClass } from 'babylonjs/Misc/typeStore';
+import { Measure } from '../measure';
+import { InputText } from "./inputText";
+import { ICanvasRenderingContext, ITextMetrics } from "babylonjs/Engines/ICanvas";
+import { PointerInfoBase } from "babylonjs/Events/pointerEvents";
+import { IKeyboardEvent } from "babylonjs/Events/deviceInputEvents";
+
+/**
+ * Enum that determines the text-wrapping mode to use.
+ */
+export enum TextWrapping_ {
```

Comment:
nit - why id you end with an underscore?

Reply:
It's an old implementation of first prototype. I removed it and reimported from the correct file.